suite-native/module-trading/src/components/general/Offline/OfflineCard.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactNode } from 'react';
+
+import { Box, Card, Pictogram, Text, VStack } from '@suite-native/atoms';
+
+export type OfflineCardProps = {
+    title: ReactNode;
+    description: ReactNode;
+    children?: ReactNode;
+};
+
+export const OfflineCard = ({ title, description, children }: OfflineCardProps) => (
```

Comment:
I would like to propose small harmonisation, because I have just found we can use `PictogramTitleHeader`

```suggestion
export const OfflineCard = ({ title, description, children }: OfflineCardProps) => (
    <Box flex={1} alignItems="center" justifyContent="center">
        <Card>
            <VStack spacing="sp24" paddingVertical="sp8">
                <PictogramTitleHeader
                    variant="warning"
                    title={title}
                    titleVariant="highlight"
                    subtitle={
                        <Text variant="hint" color="textSubdued" textAlign="center">
                            {description}
                        </Text>
                    }
                />
                {children}
            </VStack>
        </Card>
    </Box>
);
```